continuous_eval/metrics/generation_LLM_based_metrics.py

```diff
@@ -64,8 +64,12 @@ def calculate(self, question, retrieved_contexts, answer, **kwargs):
             }
 
             response = self._llm.run(prompt)
-            score_txt, reasoning = response.split("\n", 1)
-            score = bool("yes" in score_txt.lower())
+            try:
+                score_txt, reasoning = response.split("\n", 1)
+                score = float("yes" in score_txt.lower())
+            except Exception as e:
+                score = None
```

Comment:
How is it behaving the rest of the code (for example the ensemble) with the None?